lib/rails_admin/engine.rb

```diff
@@ -44,6 +34,28 @@ class Engine < Rails::Engine
       end
     end
 
+    initializer 'RailsAdmin apply configuration', after: :eager_load! do |app|
+      RailsAdmin::Config.initialize!
+
+      # Force route reload, since it doesn't reflect RailsAdmin action configuration yet
+      app.reload_routes!
+    end
+
+    initializer 'RailsAdmin precompile hook', group: :all, after: 'RailsAdmin apply configuration' do |app|
+      case RailsAdmin.config.asset_source
```

Comment:
I don't understand the problem, what did you do and what did you get?
Did you try to do `config.asset_source = :importmap` without installing `importmap-rails`?

Reply:
`config.asset_source = :importmap`. 

I believe the above refers to RailsAdmin.config.asset_source.

I believe this problem occurs because `$ bin/rails rails_admin:install` has not yet created `config/initializers/rails_admin.rb` at the stage of running `$ bin/rails rails_admin:install`.

Reply:
Ah I understood now, thanks for letting me know. Fixed by 705fd32fcea77978ed867d66696464da9b741a11 👍